supervision/dataset/core.py

```diff
@@ -181,11 +181,11 @@ def split(
             using the provided split_ratio.
 
         Args:
-            split_ratio (float, optional): The ratio of the training
+            split_ratio (float): The ratio of the training
```

Comment:
I noticed that we added types here but the actually function definition does not have type hints :/